crates/rattler_conda_types/src/repo_data/mod.rs

```diff
@@ -66,23 +84,50 @@ pub struct PackageRecord {
 
     pub features: Option<String>,
 
+    /// If this package is independent of architecture this field specifies in what way. See
+    /// [`NoArchType`] for more information.
+    #[serde(skip_serializing_if = "NoArchType::is_none")]
     pub noarch: NoArchType,
 
-    pub preferred_env: Option<String>,
-
+    /// The specific license of the package
     pub license: Option<String>,
+
+    /// The license family
     pub license_family: Option<String>,
 
-    // pub package_type: ?
+    /// The UNIX Epoch timestamp when this package was created. Note that sometimes this is specified in
+    /// seconds and sometimes in milliseconds.
     pub timestamp: Option<usize>,
-
-    pub date: Option<String>,
-
-    pub size: Option<usize>,
+    //pub preferred_env: Option<String>,
```

Comment:
maybe add a small comment about where you found those?

Reply:
I added this comment:

```rust
// Looking at the `PackageRecord` class in the Conda source code a record can also include all
// these fields. However, I have no idea if or how they are used so I left them out.
```